typescript/scheduledTasks.ts

```diff
@@ -69,17 +84,27 @@ export interface NewTask {
   swipeLeftActionId?: number
```

Comment:
actionIds needs to be a string array now, pleaaase and thank you

Reply:
`swipeLeftActionId` and `swipeRightActionId` also require this change to be a string